steps/household/household.tsx

```diff
@@ -15,7 +15,10 @@ import {
 import { ImageInput } from "../../components/ImageInput";
 import { makeSubmit } from "../../components/makeSubmit";
 import { Table } from "../../components/Table";
-import { TextAreaDetails } from "../../components/TextAreaDetails";
+import {
+  TextAreaDetails,
+  Props as TextAreaDetailsProps
```

Comment:
Let's rename them to this in the source file instead of renaming them on import.